src/components/ShoppingCart.js

```diff
@@ -34,11 +48,17 @@ const ShoppingCart = ({ cart, setCart, updateCart, isOpen, setIsOpen }) => {
     >
       <section id="cart-header">
         <h3>Your Cart</h3>
-        <button className="cart-x-button" onClick={() => setIsOpen(!isOpen)}>
-          x
-        </button>
+        <IoIosClose
+          className="cart-x-button"
+          onClick={() => setIsOpen(!isOpen)}
```

Comment:
Since the close button only closes the cart. You don't need to toggle `isOpen`. You can specify the value as false. 
So `setIsOpen(false)`.